doc/man3/SSL_set_quic_tls_cbs.pod

```diff
@@ -70,6 +70,11 @@ given SSL object I<s>, a set of callbacks are supplied in an B<OSSL_DISPATCH>
 table via I<qtdis>. The I<arg> parameter will be passed as an argument when the
 various callbacks are called.
 
+The above callbacks are invoked, as needed, by SSL_do_handshake() and SSL_read() (including
+SSL_read_ex, SSL_peek, SSL_peek_ex). Once the SSL handshake is complete the QUIC
+stack must arrange to call one of the SSL_read() variants whenever a post-handshake CRYPTO
+frame is received, the number of bytes requested should be zero.
```

Comment:
```suggestion
frame is received. The number of bytes requested may be zero.
```
This seems like a new sentence. Also I don't think the requested number of bytes *has* to be zero - it just can be. Of course we would never expect application bytes to ever be returned from such a call (so requesting zero makes sense).